### PR TITLE
[Module] Label Map module.

### PR DESCRIPTION
**Pull request by gagnonanthony**
## Describe your changes

Add label map module.

## List test packages used by your module

- `tractometry.zip`

## Checklist before requesting a review

- Create the tool:
  - [x] Edit `./modules/nf-scil/<category>/<tool>/main.nf`
  - [x] Edit `./modules/nf-scil/<category>/<tool>/meta.yml`
- Generate the tests:
  - [x] Edit `./tests/modules/nf-scil/<category>/<tool>/main.nf`
  - [x] Edit `./tests/modules/nf-scil/<category>/<tool>/nextflow.config`
  - [x] Add test data locally for tests with the fork repository
  - [x] Generate the test infrastructure and _md5sum_ for all outputs
- Ensure the syntax is correct :
  - [x] Check indentation abides with the rest of the library (don't hesitate to correct others !)
  - [x] Lint everything. Ensure your variables have good names.

### CODE REVIEW

File: modules/nf-scil/bundle/labelmap/main.nf [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    """
+    bundles=(${bundles.join(" ")})
+    centroids=(${centroids.join(" ")})
+
+    for index in \${!bundles[@]};
+        do ext=\${bundles[index]#*.}
+        bname=\$(basename \${bundles[index]} .\${ext})
+
+        if [[ -f \${centroids[index]} ]]; then
```
Comment (AlexVCaron) [inline modules/nf-scil/bundle/labelmap/main.nf:37]: Remove the `if`, I would prefer the process to fail if the centroids list doesn't have the same size.

**Review by AlexVCaron — CHANGES_REQUESTED** [review]
One minor comment, everything else is #1 !

File: modules/nf-scil/bundle/labelmap/main.nf [hunk 2]
```diff
@@ -0,0 +1,67 @@
+
```
Comment (arnaudbore) [inline modules/nf-scil/bundle/labelmap/main.nf:1]: extra lign

File: modules/nf-scil/bundle/labelmap/tests/nextflow.config [hunk 3]
```diff
@@ -0,0 +1,7 @@
+process {
+    withName: "BUNDLE_LABELMAP" {
```
Comment (arnaudbore) [inline modules/nf-scil/bundle/labelmap/tests/nextflow.config:2]: missing publishdir

File: modules/nf-scil/bundle/labelmap/meta.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+  - labels:
+      type: file
+      description: List of Nifti image. The pixel intensity corresponds to the label
+      pattern: "*.{nii,nii.gz}"
```
Comment (arnaudbore) [inline modules/nf-scil/bundle/labelmap/meta.yml:45]: be more specific

Comment (arnaudbore) [reply modules/nf-scil/bundle/labelmap/meta.yml:45]: .nii.gz no need for nii alone

File: modules/nf-scil/bundle/labelmap/meta.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      description: List of trk. The streamline color corresponds to the label
+      pattern: "*.{trk}"
+
+  - distances_trk:
+      type: file
+      description: List of Nifti image for each bundle. The streamline color corresponds to the distance to the centroid
+      pattern: "*.{trk}"
+
+authors:
+  - "@gabknight"
```
Comment (arnaudbore) [inline modules/nf-scil/bundle/labelmap/meta.yml:63]: add your name

**Review by arnaudbore — COMMENTED** [review]
couple of comments

**Review by arnaudbore — APPROVED** [review]
small comment otherwise LGTM !

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**